frontend/components/elements/StoryItem.vue

```diff
@@ -124,42 +123,34 @@
                 </BDropdown>
             </td>
 
-            <td v-if="isView('backlog')">
+            <td v-if="isView(['backlog', 'planning-backlog', 'planning-sprint'])">
                 <BButton 
                     size="sm" 
-                    variant="primary" 
-                    title="Edit" 
+                    variant="outline-primary"
+                    title="Edit"
+                    class="mb-1"
                     @click="startEditing">
 
                     <VIcon name="pencil"/>
+                    Edit
                 </BButton>
 
                 <BButton 
                     size="sm" 
-                    variant="danger" 
+                    variant="outline-danger"
                     title="Delete" 
                     v-confirm="{ action: remove, text: 'Are you sure you want to delete this story?' }">
```

Comment:
There is no 7x confirmation loop for deleting a story anymore. :+1:

Reply:
whoop!